README.rst

```diff
@@ -47,7 +47,7 @@ submitted directly to the `bug tracker
 <https://github.com/pypa/setuptools/issues>`_.
 
 
-Code of Conduct
+Code of Conductt
```

Comment:
Don't make changes unrelated to the PR title, it only adds confusion. Also, this just introduces a typo.
```suggestion
Code of Conduct
```